src/benchq/resource_estimation/of_modified.py

```diff
@@ -0,0 +1,222 @@
+#   Copyright 2017 The OpenFermion Developers
+#   Modifications copyright 2023 Zapata Computing, Inc. for compatibility reasons.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#coverage:ignore
```

Comment:
Let's remove the `#coverage:ignore`, would be good to account for this code in our coverage analysis.

Reply:
Could you please tell what is coverage analysis ?

Reply:
Code coverage tells us how much of the code is accessed by our tests. This metric lets us track if our code is well tested. We typically required a coverage level of at least 75% for all our repos.